MANIFEST.in

```
@@ -0,0 +1,3 @@
+include kinkernel/py.typed
+include README.md
+include LICENSE
```

end of file black space is missing
```suggestion
include LICENSE

```